psycopg3_c/psycopg3_c/_psycopg3.pyi

```diff
@@ -22,6 +22,10 @@ class Transformer(proto.AdaptContext):
     @property
     def adapters(self) -> AdaptersMap: ...
     @property
+    def make_row(self) -> Optional[proto.RowMaker]: ...
+    @make_row.setter
+    def make_row(self, row_maker: proto.RowMaker) -> None: ...
```

Comment:
It seems the setter might take a None as well?

Reply:
That's correct. Fixed in dda10aabd4556a4121f89f8c3a9678fb21bdb1cd.